Mathlib/Data/Fintype/Sum.lean

```diff
@@ -61,8 +60,7 @@ theorem image_subtype_ne_univ_eq_image_erase [Fintype α] (k : β) (b : α → 
     exact ⟨⟨a, ne_of_mem_erase hi⟩, mem_univ _, rfl⟩
 #align image_subtype_ne_univ_eq_image_erase image_subtype_ne_univ_eq_image_erase
 
-open Classical in
```

Comment:
This pattern is probably bad for any `theorem`s in mathlib 4; at some point it would be worth looking for other occurrences; but not in this PR